src/pages/welcome.html

```diff
@@ -52,13 +54,13 @@ <h2 class="ff-Met document-subhead">Using Private Network</h2>
           </figure>
 
           <p>Select the icon and you'll be asked to sign in with a Firefox account. If you don’t already have a Firefox account, you can also create one by clicking <strong>Continue</strong>.</p>
-          <p>Once you’re signed in, you’ll see a green check on the Private Network icon. This means it is on. Use Firefox as usual, and your online activity will be encrypted and sent through a proxy service provided by our trusted partner Cloudflare.</p>
+          <p>Once you’re signed in, you'll receive 12 passes a month of Firefox Private Network, each valid for one hour. To claim a pass, simply turn Private Network on. Use Firefox as usual, and your browsing will be encrypted and sent through a proxy service provided by our trusted partner Cloudflare. Passes expire after one hour, even if you turn Private Network off. You’ll receive 12 new passes at the beginning of each month.</p>
```

Comment:
In the first sentence, the apostrophes are different on “you’re” and “you’ll”. The you’ll should use smart quotes.